source/administration/config-settings.rst

```diff
@@ -2099,6 +2099,8 @@ Segment Write Key
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 For deployments seeking additional tracking of system behavior using Segment.com, you can enter a Segment WRITE_KEY using this field. This value works like a tracking code and is used in client-side Javascript and will send events to Segment.com attributed to the account you used to generate the WRITE_KEY.
 
+.. note:: This field will be removed from config.json in Mattermost 3.7, released on March 16th, 2017.
```

Comment:
Does it make sense to say "releasing" or "which will be released" instead of "released" to keep the tense consistent?